src/utils/Multicall.ts

```diff
@@ -26,6 +26,7 @@ const NON_DETERMINISTIC_MULTICALL_ADDRESSES = {
 const DETERMINISTIC_MULTICALL_CHAINS = [
   CHAIN_IDs.ALEPH_ZERO,
   CHAIN_IDs.ARBITRUM,
+  CHAIN_IDs.DOCTOR_WHO,
```

Comment:
@gsteenkamp89 This isn't necessary for OP stack chains (see line 41 below).

Reply:
ah its a preinstall, got it 👍